geolocator_apple/pubspec.yaml

```
@@ -20,7 +20,7 @@ dependencies:
   flutter:
     sdk: flutter
 
-  geolocator_platform_interface: ^4.0.0
+  geolocator_platform_interface: ^4.0.0+1
```

This should be version 4.0.3:
```suggestion
  geolocator_platform_interface: ^4.0.3
```